README.md

```diff
@@ -118,7 +118,7 @@ based on the following:
 
 ## Support
 
-Got questions?
+Have you got any questions?
```

Comment:
Please revert this.

Reply:
Sure. Done.
Can you shortly explain the reasoning behind this?

Reply:
I want to have it short, simple and consistent.

Reply:
Thanks. Change applied.